auth/middleware.go

```diff
@@ -51,33 +52,66 @@ func sessionCallbacks(request *http.Request, writer http.ResponseWriter) *http.R
 }
 
 func reqisterUser(request *http.Request, writer http.ResponseWriter, db *gorm.DB) *http.Request {
-	if token, err := getToken(request); err == nil {
-		device := &model.Device{}
-		if db.Where("token = ?", token).Find(device).RecordNotFound() {
-			log.Info().Str("token", token).Msg("no device with token found")
+	token, err := getToken(request)
+	if err != nil {
+		msg := "Issue fetching token"
+		log.Info().Msg(msg)
+		http.Error(writer, msg, 400)
```

Comment:
I think this is problematic @jmattheis.

I'm seeing output like this (on http://localhost:3030/):
```
Issue fetching token
<!doctype html><html lang="en"><head><meta charset="utf-8"/>...</head><body>...</body></html>
```
Do you see the same issue on your end?

Reply:
Yeah, fixed in the last commit.